services/src/services/errorTelemetry/errorTelemetry.hooks.js

```diff
@@ -31,7 +35,8 @@ class ErrorTelemetryHooks {
                 throw new errors.BadRequest(`Missing required field '${field}'`);
               }
             });
-          }
+          },
+          ensureMatchingUUID
```

Comment:
Please make this the first hook. Whenever we have authenticated APIs like this which take a `uuid` parameter, I would like the ensure hook to be first in the list to avoid any other hooks needing to do potentially unnecessary work (and resulting in work being done on an invalid request, there are some security concerns that raises for me which don't necessarily apply in this context)